docs/installation.rst

```diff
@@ -20,6 +20,35 @@ There are different ways to install |project_name|:
     have the latest code or use revision control (each release is
     tagged).
 
+.. _installation-requirements:
+
+Pre-Installation Considerations
+-------------------------------
```

Comment:
I'm open to suggestions ... looking through the existing sections, there doesn't seem to be an existing section where this would better fit.

There isn't currently a requirements section. "Installation" is the first major section of the documentation.  Logically, before installing an app, one should understand whether their system meets the requirements of the app.

I've found that a lot of this information is already available in different places spread out in the docs... but there isn't one location where all this information is pulled together.

It can possibly be shortened and added to the FAQs, but I'm not sure that would have helped in the case of the original bug report.  Maybe #2080 can be resolved with a list of file systems explicitly *NOT* supported?  I think a list of *supported* file systems would be rather long and cumbersome to maintain. Presumably newer file systems will be supported, while a handful of old file systems (e.g. Fat16) may work but aren't recommended?